WooCommerce/Classes/ViewRelated/Orders/Order Creation/Taxes/NewTaxRateSelectorView.swift

```diff
@@ -37,65 +37,46 @@ struct NewTaxRateSelectorView: View {
                     .footnoteStyle()
                     .multilineTextAlignment(.leading)
                     .padding([.leading, .trailing], Layout.generalPadding)
-                    .padding(.bottom, Layout.taxRatesSectionTitleBottomPadding)
+                    .padding([.top, .bottom], Layout.taxRatesSectionTitleVerticalPadding)
 
                 Divider()
 
-                ForEach(viewModel.demoTaxRates, id: \.title) { taxRate in
-                    HStack {
-                        Button(action: { }) {
-                            AdaptiveStack(horizontalAlignment: .leading, spacing: Layout.generalPadding) {
-                                Text(taxRate.title)
-                                    .foregroundColor(Color(.text))
-                                    .multilineTextAlignment(.leading)
-                                    .frame(maxWidth: .infinity, alignment: .leading)
-
-                                Text(taxRate.value)
-                                    .foregroundColor(Color(.text))
-                                    .multilineTextAlignment(.trailing)
-                                    .frame(width: nil, alignment: .trailing)
+                switch viewModel.syncState {
+                    case .results:
+                        RefreshableInfiniteScrollList(isLoading: viewModel.shouldShowBottomActivityIndicator,
+                                                      loadAction: viewModel.onLoadNextPageAction,
+                                                      refreshAction: { completion in
+                            viewModel.onRefreshAction(completion: completion)
+                        }) {
+                            ForEach(viewModel.taxRateViewModels, id: \.id) { viewModel in
+                                TaxRateRow(viewModel: viewModel)
+                                Divider()
```

Comment:
Small stylistic thought: perhaps the divider should be part of the row? Or a viewModifier on it? Not a big deal.

Reply:
That makes sense. It should probably be a modifier (or default behaviour) on the List then, I think that's how it works with the built in List.